app/src/main/res/raw/menu.json

```diff
@@ -1,6 +1,25 @@
 {
 	"version": 2018102402,
 	"schedules": [
+		{
+			"title": "35th Chaos Communication Congress",
+			"metadata": {
+				"links": [
+					{
+						"url": "https://events.ccc.de/congress/2018/wiki/index.php/Main_Page",
+						"title": "Wiki"
+					},
+					{
+						"title": "Weblog",
+						"url": "https://events.ccc.de/"
+					}
+				]
+			},
+			"url": "https://fahrplan.events.ccc.de/congress/2018/Fahrplan/schedule.xml",
+			"start": "2017-12-26",
```

Comment:
This looks like the wrong year (same with end date, and possibly with version field).

Reply:
whoops, the typical copy-pasta error :( thanks for spotting!